unitstatushandler/unitstatushandler.go

```diff
@@ -461,77 +463,73 @@ func (descriptor *statusDescriptor) getVersion() (version string) {
 	}
 }
 
-func (instance *Instance) updateUnitConfigStatus(unitConfigInfo cloudprotocol.UnitConfigStatus) {
+func (instance *Instance) updateUnitConfigStatus(status cloudprotocol.UnitConfigStatus) {
 	instance.statusMutex.Lock()
 	defer instance.statusMutex.Unlock()
 
 	log.WithFields(log.Fields{
-		"status":  unitConfigInfo.Status,
-		"version": unitConfigInfo.Version,
-		"error":   unitConfigInfo.ErrorInfo,
+		"status":  status.Status,
+		"version": status.Version,
+		"error":   status.ErrorInfo,
 	}).Debug("Update unit config status")
 
-	instance.processUnitConfigStatus(unitConfigInfo)
+	instance.processUnitConfigStatus(status)
 	instance.statusChanged()
 }
 
-func (instance *Instance) processUnitConfigStatus(unitConfigInfo cloudprotocol.UnitConfigStatus) {
-	instance.updateStatus(&instance.unitConfigStatus, statusDescriptor{&unitConfigInfo})
+func (instance *Instance) processUnitConfigStatus(status cloudprotocol.UnitConfigStatus) {
+	instance.updateStatus(&instance.unitStatus.unitConfig, statusDescriptor{&status})
 }
 
-func (instance *Instance) updateComponentStatus(componentInfo cloudprotocol.ComponentStatus) {
+func (instance *Instance) updateComponentStatus(status cloudprotocol.ComponentStatus) {
 	instance.statusMutex.Lock()
 	defer instance.statusMutex.Unlock()
 
 	log.WithFields(log.Fields{
-		"id":            componentInfo.ComponentID,
-		"status":        componentInfo.Status,
-		"vendorVersion": componentInfo.Version,
-		"error":         componentInfo.ErrorInfo,
+		"id":            status.ComponentID,
+		"status":        status.Status,
```

Comment:
Let's log component type as well
```suggestion
		"type":        status.ComponentType,
```

Reply:
done